EasyPost.Integration/Basics.cs

```diff
@@ -11,7 +11,7 @@ public class Basics
     ///     If this test can be compiled, then the response objects have public constructors.
     /// </summary>
     [Fact, Testing.Access, Testing.Compile]
-    public void TestUserCanLocallyConstructResponseObject()
+    public void UserCanLocallyConstructResponseObject()
```

Comment:
question: why remove the `Test` prefix? Keeping it quickly helps scanning eyes know exactly what it is

Reply:
They're all tests, so it becomes a redundant word

Reply:
We'll agree to disagree